src/assets/css/components/popup.css

```diff
@@ -0,0 +1,50 @@
+.popup-container .overlay {
```

Comment:
We should use BEM so here we should only use one class so instead .popup-container .overlay we should use something like .popup__overlay (just this, without .popup-container).